Formula/prototool.rb

```diff
@@ -0,0 +1,31 @@
+class Prototool < Formula
+  desc "Your Swiss Army Knife for Protocol Buffers"
+  homepage "https://github.com/uber/prototool"
+  url "https://github.com/uber/prototool/archive/v1.1.0.tar.gz"
+  sha256 "7759bd4db9a907d3dea4abb34ff18ddbc814f4de56d43aca8d2f402e6fd623fa"
+
+  depends_on "automake" => :build
+  depends_on "glide" => :build
+  depends_on "go" => :build
+
+  def install
+    ENV["GOPATH"] = buildpath
+    prototoolpath = buildpath/"src/github.com/uber/prototool"
+    prototoolpath.install buildpath.children
+    cd prototoolpath do
+      system "make", "brewgen"
+      cd "brew" do
```

Comment:
Any chance of some kind of `make install` step rather than manually placing everything? We try to avoid manual placement because it has a splendid tendency to become problematic over time.

Reply:
What would you suggest specifically? The helper functions here know the locations to install each item, produce messages such as “bash completion has been installed to”, etc, there is no way to move this logic into Prototool as I see it.

If there’s a specific recommendation, more happy to take it, otherwise would surmise we leave this as is. Note it’s patterned off of many existing Formulas - not that we wouldn’t always strive to do better, but I think this is actually pretty maintainable and consistent from my (novice with homebrew-core) view.